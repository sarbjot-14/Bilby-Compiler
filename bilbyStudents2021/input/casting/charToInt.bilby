main{
	
	imm first := #A;
	print [first as int] $n;
	
	imm second := #!;
	print [second as int] $n;
	%33
	
	imm third := #};
	print [third as int] $n;
	%125
	
	
	
	
}