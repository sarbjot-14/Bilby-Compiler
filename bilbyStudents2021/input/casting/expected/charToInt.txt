65
33
125
